job-tutorial/README.md

```diff
@@ -130,6 +130,7 @@ Below we show a complete list of environment variables accessible in a Docker co
 
 | Environment Variable Name          | Description                              |
 | :--------------------------------- | :--------------------------------------- |
+| PAI_DEFAULT_HDFS_URI               | Default HDFS URI in PAI                  |
```

Comment:
PR 276 has a duplicated fix like this one, but also have another fix to add task GPU count. Could you cherry pick the GPU count comment into your PR to merge them please?

Reply:
pls refer to `PAI_TASK_GPU_NUM` https://github.com/Microsoft/pai/blob/master/rest-server/src/templates/dockerContainerScript.mustache#L47

Reply:
should we change PAI_DEFAULT_HDFS_URI to PAI_DEFAULT_FS_URI

Reply:
updated